src/replay/components/player/VideoStreamer/HlsjsVideoStreamer/hlsjsBitrateManager.js

```diff
@@ -10,13 +10,23 @@ declare class Object {
 }
 
 type PropsWithInitial = {
-  initialPlaybackProps?: InitialPlaybackProps
+  initialPlaybackProps?: InitialPlaybackProps,
+  configuration?: any
 };
 
 function getBitrateAsKbps(level: HlsjsQualityLevel) {
   return (level && Math.ceil(level.bitrate / 1000)) || 0;
 }
 
+function manualbitrateSwitch(hls, configuration, switchTrackId) {
```

Comment:
As pointed out for the Shaka implementation, please change into a capital B in this function name.

Reply:
Resolved this function name here as well.